coral/src/app/features/components/AclDetailsModalContent.tsx

```diff
@@ -56,7 +56,8 @@ const TopicDetailsModalContent = ({ request }: DetailsModalContentProps) => {
         <dd>{topicname}</dd>
       </Box.Flex>
 
-      <GridItem colSpan={"span-2"}>
+      <Grid.Item />
```

Comment:
I don't think this closing tag is closing anything?Maybe a leftover? It should probably be deleted.